query/tests/it/servers/http/clickhouse_handler.rs

```diff
@@ -257,7 +257,7 @@ async fn test_insert_format_ndjson() -> PoemResult<()> {
         let (status, body) = server
             .post("insert into table t1 format JSONEachRow", &body)
             .await;
-        assert_eq!(status, StatusCode::INTERNAL_SERVER_ERROR);
+        assert_eq!(status, StatusCode::BAD_REQUEST);
         assert_error!(body, "column a");
```

Comment:
what does error message "column a" mean?

Reply:
Means error contains this string?  It's written by you 3 months ago...

Reply:
leave it alone, I will fix it  tomorrow